src/client/auth/auth.go

```diff
@@ -47,22 +45,6 @@ func ParseScope(s string) (Scope, error) {
 	return Scope_NONE, fmt.Errorf("unrecognized scope: %s", s)
 }
 
-// In2Out converts an incoming context containing auth information into an
-// outgoing context containing auth information, stripping other keys (e.g.
-// for metrics) in the process. If the incoming context doesn't have any auth
-// information, then the returned context won't either.
-func In2Out(ctx context.Context) context.Context {
```

Comment:
I'm missing something. Why is this no longer needed?

Reply:
It's no longer called :-). Basically, when you call `pachClient.WithCtx(ctx)`, it does its own transformation of the incoming ctx before copying it to the new client struct. All of Pachyderm's code now uses that abstraction instead

Reply:
:+1: